MultiplayerMod/Multiplayer/Objects/Reference/GridReference.cs

```diff
@@ -18,7 +18,7 @@ public class GridReference : GameObjectReference {
     public GridReference(GameObject gameObject) {
         var extension = gameObject.GetComponent<GameObjectExtension>();
         Cell = Grid.PosToCell(gameObject);
-        Layer = extension?.GridLayer ?? 0;
+        Layer = extension != null ? extension.GridLayer : 0;
```

Comment:
Not sure if it's required for our operations.

Reply:
Done